openpype/modules/deadline/plugins/publish/collect_pools.py

```diff
@@ -9,11 +19,12 @@
 
 class CollectDeadlinePools(pyblish.api.InstancePlugin,
                            OpenPypePyblishPluginMixin):
-    """Collect pools from instance if present, from Setting otherwise."""
+    """Collect pools from instance or attributes, from Setting otherwise."""
 
     order = pyblish.api.CollectorOrder + 0.420
     label = "Collect Deadline Pools"
-    families = ["rendering",
+    families = ["render",
```

Comment:
Note that this will now show for all render families, including:

- Create Render in Unreal
- Create Render in TV Paint
- Create Movie Batch in Tray Publisher
- Create Render in Harmony
- Create Saver in Fusion (which it already did, because it had `render.farm` I believe?)
- Create Render in After Effects
- Create Render in Nuke

Is it correct that the attribute definitions should show and thus also the collector should run for all of those?

Reply:
I'll check `get_publish_families`. 
I am on the fence with this, I don't like this approach, but don't know better one. Feedback welcomed.
I think this comes together with issue I created https://github.com/ynput/OpenPype/issues/6078

Reply:
Maybe add the below the first plugin?
```python
class CollectDeadlinePoolsAE(CollectDeadlinePools):
    families = ["render"]
    hosts = ["aftereffect"]
```

Reply:
The issue of not showing Pools fields is not only about AE, I think it is there also for Nuke, but nobody noticed it yet, imho.

The issue is to show this fields for `farm rendered` instances only.

(BTW for Fusion it is not there either:
![image](https://github.com/ynput/OpenPype/assets/4457962/9ebe1150-92d6-41fc-a2ae-559d63f19209)
)

Reply:
> The issue of not showing Pools fields is not only about AE, I think it is there also for Nuke, but nobody noticed it yet, imho.

Possible, but by adding `"render"`, it will also show for all other hosts where it should not be shown.

Reply:
Added hosts filter to be more precise.